docs/standard-json-format.rst

```diff
@@ -0,0 +1,112 @@
+.. index:: standard.json file format
```

Comment:
Can we do this in its own PR?

Reply:
that's what i initially wanted. I'll move it out, sure.